genai_cookbook/nbs/3-deep-dive-chain.md

```diff
@@ -1,10 +1,10 @@
-## Retrieval, augmentation, and generation (aka RAG Chain)
+## Retrieval, augmentation, and generation (aka RAG Agent)
```

Comment:
Should we just call this "building an agent with a retrieval tool" and reference the RAG pattern again? or link to databricks docs about that

Reply:
Actually, I guess RAG agent is ok - I do see it referenced elsewhere e.g. https://langchain-ai.github.io/langgraph/tutorials/rag/langgraph_agentic_rag/